packages/orchestration/src/facade.js

```diff
@@ -87,9 +79,10 @@ export const makeOrchestrationFacade = ({
     const hostFn = asyncFlow(subZone, 'asyncFlow', guestFn);
 
     const orcFn = (...args) =>
-      // TODO remove the `when` after fixing the return type
-      // to `Vow<HostReturn>`
-      when(hostFn(wrappedOrc, wrappedCtx, ...args));
+      // @ts-expect-error cast
+      hostFn(wrappedOrc, wrappedCtx, ...args);
```

Comment:
I'm not really a fan of error suppressions on whole lines that could have multiple errors.